src/pnpm/doPnpmPublishVersions.ts

```diff
@@ -0,0 +1,36 @@
+import { spawnSync } from "node:child_process";
+
+/**
+ * Bump all changed packages to version {newVersion}
+ * @param since selects all the packages changed since the specified commit/branch/tag
+ * @param tag publishes the package with the given tag
+ * @param branch primary branch of the repository which is used for publishing the latest changes
+ * @param forcePackages also bump the version in those packages
+ * @param excludePackages filter out the specified packages/directory from the output
+ */
+export default function (
+  since: string,
+  tag: string = "latest",
+  branch: string = "main",
```

Comment:
I would not put `main` as a default arg but either: 
- `master` until Git SCM and PNPM change their defaults.
- no default, require the param.

There is of course a factor of inclusivity at stake, but functionality and respect of standard, in this particular instance, are more important.